monitoring/mock_uss/scdsc/routes_scdsc.py

```diff
@@ -65,6 +70,21 @@ def make_uss_report():
 
     return flask.jsonify({"message": "Not yet implemented"}), 500
 
+
+if "msgsigning" in SERVICES:
```

Comment:
I would [expect](https://github.com/interuss/dss/tree/master/monitoring/uss_qualifier/suites/faa/uft/design#positive-tests--) this endpoint to be hosted as part of a [new](https://github.com/interuss/dss/pull/922#discussion_r1042878898) `msgsigning` [service capability](https://github.com/interuss/dss/blob/17c4f01bd787014bcba0cc6942131192398834ae/monitoring/mock_uss/__init__.py#L25) of mock_uss (similar to [`geoawareness`](https://github.com/interuss/dss/tree/master/monitoring/mock_uss/geoawareness), [`riddp`](https://github.com/interuss/dss/tree/master/monitoring/mock_uss/riddp), [`ridsp`](https://github.com/interuss/dss/tree/master/monitoring/mock_uss/ridsp), and [`scdsc`](https://github.com/interuss/dss/tree/master/monitoring/mock_uss/scdsc)).